ticdc/ticdc-overview.md

```diff
@@ -111,6 +111,12 @@ Solutions: Use the `cdc` executable file corresponding to the TiCDC cluster vers
 >
 > The above issue exists only when `cdc cli` is v5.0.0-rc. Other v5.0.x `cdc cli` tool can be compatible with v4.0.x clusters.
 
+### Compatibility with temporary tables
+
+Since v5.3.0, TiCDC supports [global temporary tables](/temporary-table.md#global-temporary-tables). Replicating global temporary tables using TiCDC of a version earlier than v5.3.0 causes table definition error.
+
+If the upstream cluster contains a global temporary table, the downstream TiDB cluster is expected to be v5.3.0 or later. Otherwise, an error occurs in the replication.
```

Comment:
```suggestion
Since v5.3.0, TiCDC supports [global temporary tables](/temporary-table.md#global-temporary-tables). Replicating global temporary tables to the downstream using TiCDC of a version earlier than v5.3.0 causes table definition error.

If the upstream cluster contains a global temporary table, the downstream TiDB cluster is expected to be v5.3.0 or a later version. Otherwise, an error occurs during the replication process.
```